### ISSUE TITLE
Root password in krill install showed as changed but not in install

### ISSUE THREAD

**Issue by rreyn331** [pieroproietti/penguins-eggs-legacy#174]
on a naked duvean install with runit init and doas, it might not matter.  ISO created that when installed and  says password used for user is the same as root but root password when booted into system has default password.  There was no option to change root password during install yet it says it has been changed
![20221218_112502](https://user-images.githubusercontent.com/110791312/208312501-b037f79e-39b2-4023-a8c4-0bbfce4ed231.jpg)
![Uploading 20221218_112252.jpg…]()

**Comment by pieroproietti** [comment]
Yes, you are right. How I have a bit spare time I will fix it. Thanks

**Comment by ghost** [comment]
Hello,
and sry for commenting on this old Issue.
But i habe exactly the same problem (on the newest Version)
Under the User Step the passwort is showing as changed.
But under summary is the default: evolution Password

**Comment by pieroproietti** [comment]
Hi Tobias,

this is definitely my fault! I'm used to install with the command: `eggs install -un` unattended, and change later - if I need - users, passwords, keyboards and so on.

I did a test, just now before to answer, I saw user and password are OK, krill don't ask for root password at all and leave it unchanged (evolution).

As far I have a bit time I will put hands on krill, and see to solve this problem.
